src/test/java/signUpTesting/VerifySignUpLink.java

```diff
@@ -0,0 +1,23 @@
+package signUpTesting;
+
+import org.openqa.selenium.*;
+import org.testng.annotations.*;
+import utils.DriverSetUp;
+import org.testng.*;
+
+import static org.hamcrest.Matchers.*;
+import static org.hamcrest.MatcherAssert.*;
+
+public class VerifySignUpLink extends DriverSetUp {
+
+    @Test
+    public void verifySignUpLinkPresent() {
+        driver.findElement(By.xpath("//a[text()='Sign up']")).isDisplayed();
+    }
+
+    @Test
+    public void verifySignUpLinkCorrectTitle(){
+        String signUpLink = driver.findElement(By.xpath("//a[text()='Sign up']")).getText();
```

Comment:
it might be a good idea to use Page object pattern

Reply:
Will be added next week.